src/ert/run_models/run_arguments.py

```diff
@@ -15,6 +15,7 @@ class SimulationArguments:
 class SingleTestRunArguments(SimulationArguments):
     current_case: str
     target_case: Optional[str] = None
+    analysis_module: str = "Single test"
```

Comment:
analysis_module is a bit overloaded, so suggest we rename this ensemble_type instead